packages/bvaughn-architecture-demo/components/console/MessageHoverButton.module.css

```diff
@@ -29,6 +31,7 @@
 .ConsoleMessageHoverButtonIcon {
   width: 1.25rem;
   height: 1.25rem;
+  padding: 0.0625rem;
```

Comment:
This looks sketchy.

Reply:
Half of 0.125 😄 I'll work on getting a spacing scale in so these values are at least tied to a system. We'll need these smaller values though since we're dealing a with pretty condensed UI.